### PR TITLE
Ui improvements

### PR DESCRIPTION
**Pull request by Zales0123**
Based on https://github.com/Sylius/AdminOrderCreationPlugin/pull/7

Before:
before

After:
zrzut ekranu 2018-05-23 o 09 31 46
zrzut ekranu 2018-05-23 o 09 31 55
zrzut ekranu 2018-05-23 o 09 32 08

### CODE REVIEW

**Comment by pamil** [comment]
Could you rebase this PR?

**Comment by Zales0123** [comment]
Ay

File: src/Resources/views/orderCreate.html.twig [hunk 1]
```diff
@@ -19,14 +19,26 @@
...
         {{ form_start(form, {'action': path('sylius_admin_order_creation_order_create', { 'customerEmail': app.request.attributes.get('customerEmail') }), 'attr': {'class': 'ui loadable form', 'novalidate': 'novalidate'}}) }}
 
-        <div id="items">
-            {{ form_row(form.items) }}
+        <div class="ui styled fluid accordion">
+            <div class="title active"><i class="dropdown icon"></i> {{ 'sylius.ui.items'|trans }}</div>
+            <div id="items" class="content active">
+                {{ form_row(form.items) }}
+            </div>
+            <div class="title"><i class="dropdown icon"></i> {{ 'sylius.ui.shipping_address'|trans }} & {{ 'sylius.ui.billing_address'|trans }}</div>
```
Comment (pamil) [inline src/Resources/views/orderCreate.html.twig]: `&` -> `&amp;`

File: src/Resources/views/orderCreate.html.twig [hunk 2]
```diff
@@ -19,14 +19,26 @@
...
+            <div class="content">
+                <div class="ui horizontal segments">
+                    <div class="ui segment">{{ form_row(form.shippingAddress) }}</div>
+                    <div class="ui segment">{{ form_row(form.billingAddress) }}</div>
+                </div>
+            </div>
+            <div class="title"><i class="dropdown icon"></i> {{ 'sylius.ui.shipments'|trans }} & {{ 'sylius.ui.payments'|trans }}</div>
```
Comment (pamil) [inline src/Resources/views/orderCreate.html.twig]: Same here

File: tests/Behat/Page/Admin/OrderCreatePage.php [hunk 3]
```diff
@@ -112,4 +120,23 @@ private function getItemWithProductSelected(int $productId): NodeElement
...
+    {
+        $tab = $this->getDocument()->find('css', sprintf('.title:contains("%s")', $tabName));
+
+        if ($tab->hasClass('active')) {
+            return;
+        }
+
+        $tab->click();
+
+        $this->getDocument()->waitFor(10, function () use ($tabName) {
```
Comment (bartoszpietrzak1994) [inline tests/Behat/Page/Admin/OrderCreatePage.php]: Since builds on travis are fixed, can you try reducing this amount of time?

Comment (Zales0123) [reply tests/Behat/Page/Admin/OrderCreatePage.php]: Fixed

**Pull request merged**